pkgs/kde/plasma/plasma-workspace/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  mkKdeDerivation,
+  substituteAll,
+  xorg,
+  pkg-config,
+  spirv-tools,
+  qtsvg,
+  qtwayland,
+  libcanberra,
+  libqalculate,
+  pipewire,
+  breeze,
+  qttools,
+  qqc2-breeze-style,
+}:
+mkKdeDerivation {
+  pname = "plasma-workspace";
+
+  patches = [
+    (substituteAll {
+      src = ./tool-paths.patch;
+      xmessage = "${lib.getBin xorg.xmessage}/bin/xmessage";
+      xsetroot = "${lib.getBin xorg.xsetroot}/bin/xsetroot";
+      qdbus = "${lib.getBin qttools}/bin/qdbus";
```

Comment:
Would be nice to use lib.getExe here

Reply:
It's not the only binary in qttools.

Reply:
I must admit I only looked at the other two but fine to keep as is.

Reply:
Those don't have `meta.mainProgram` I'm pretty sure

Reply:
There is an alternative function `lib.getExe'` which takes an executable as a second parameter. So `lib.getExe' qttools "qdbus"` would be equivalent.